Makefile

```diff
@@ -1,27 +1,28 @@
-ifeq ($(shell uname), Linux)
 ifeq ($(MPI), 1)
-	FC = mpif90
```

Comment:
I think that this removal of the FC variable is causing the continuous integration travis build to fail. Perhaps you could take a look to see if .travis.yml also needs to be updated (I think this would involve adjusting the build.sh script so it uses your new command, rather than make)? It also no longer works if we try to build it the old way, i.e. using make.